### ISSUE TITLE
Artifacts have a collison box

### ISSUE THREAD

**Issue by Vincent243** [Iskallia/Vault-public#562]
Artifacts have a collison box, you can get stuck when walking beside a block where an artifact is placed

**Steps to reproduce:**
1.Place and artifact on a block
2.Then break the block

**Expected behavior**
Notice you can't walk through the artifact

**Info:**
- World Type: Single Player
- The_Vault Version: 1.6.2

**Comment by joshuaepstein** [comment]
I don't think that this is a game breaking bug so it probably won't get looked at
